Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04
+
+ADD . /app
+WORKDIR /app
+
+RUN apt-get update && \
+    apt-get install python3 python3-venv python3-dev libgeoip-dev build-essential -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
to be honest I see that we can use docker for prod and in this case we don't need extra provisioning, I want propose do refactoring what you describing in separate PR as I don't have good vision how it should be in case of dev can be really separate machines with different package manager or way to install python

Reply:
Running PostrgreSQL is container is a bad idea. `docker-compose` is not a container orchestration tool, which works on a single machine only, and in the end you will need to use Kubernetes or Ansible anyway.

Reply:
There are a few points:
- we can drop postgres db and recreate it from OSM dump, so in bad case we can lose availability, but not data. For our case using container looks ok TBH
- we don't need kubernetes for one metal node
- it nice to describe what we trying solve and what pros and cons we can have with different proposed solutions